subsys/zephyrbt/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ zephyr_sources_ifdef(CONFIG_ZEPHYR_BEHAVIOUR_TREE
 )
 
 zephyr_sources_ifdef(CONFIG_ZEPHYR_BEHAVIOUR_TREE_NODE_CONTEXT
+	zephyrbt_decorator_delay.c
```

Comment:
I believe the correct name is Sleep. You can change the name of the decorator or change the code generation to adjust the code of the decorator when generating it.

Reply:
https://www.behaviortree.dev/docs/nodes-library/DecoratorNode#delay